pkg/mapper/configmap/configmap_test.go

```diff
@@ -170,7 +196,7 @@ func TestLoadConfigMap(t *testing.T) {
 
 	watcher.Add(&core_v1.ConfigMap{ObjectMeta: meta, Data: data})
 
-	time.Sleep(2 * time.Millisecond)
+	time.Sleep(2 * time.Second)
```

Comment:
sleeping in a test could cause flakiness. I imagine the change was required because the tests failed otherwise. Probably in a different PR and time, we could remove dependency on sleep in test.

Reply:
This test has been flaky.  Let's just merge this change in a separate PR first as well, and we can rebase this one on it.

Reply:
Yeah no idea what's up with this but the test was much less flaky after increasing the time. I don't have any plans to fix it but agree it needs fixing

Reply:
I'd prefer to leave it in this PR so we end up with less git operations.
I'll give it its own commit when I rebase as a compromise